packages/calcite-components/src/components/menu-item/menu-item.scss

```diff
@@ -1,3 +1,23 @@
+@import "~@esri/calcite-design-tokens/dist/scss/core";
+
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-menu-item-text-color: Specifies the text color of the component.
+ * @prop --calcite-menu-item-text-color-hover: Specifies the text color of the component when hover.
+ * @prop --calcite-menu-item-text-color-active: Specifies the text color of the component when active.
+ * @prop --calcite-menu-item-background-color: Specifies the background color of the component.
+ * @prop --calcite-menu-item-background-color-hover: Specifies the background color of the component when hover.
+ * @prop --calcite-menu-item-background-color-active: Specifies the background color of the component when active.
+ * @prop --calcite-menu-item-dropdown-icon-color: Specifies the color of the dropdown icon when sub-menu items are slotted.
+ * @prop --calcite-menu-item-dropdown-icon-color-active: Specifies the color of the dropdown icon when active.
+ * @prop --calcite-menu-item-breadcrumb-icon-color: Specifies the color of the breadcrumb icon when sub-menu items are slotted.
+ * @prop --calcite-menu-item-sub-menu-border-color: Specifies the border color of sub-menu.
+ * @porp --calcite-menu-item-sub-menu-background-color: Specifies the background color of sub-menu.
```

Comment:
I think we can use the same styling for all menu items, if possible.

Reply:
we might need the `border-color` though.